src/styles/_catalog.scss

```diff
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
- These lines of code need some empty lines.
- Do not repeat yourself, create a variable for card width